### DISCUSSION TITLE
Themeing Support

### DISCUSSION THREAD

**Discussion by njmdev03** [pelican-dev/panel#32] [Ideas]
It is possible that this is handeled through the planned addons already, but it would be nice to have a style of addons just for themeing/skinning the panel.

**Comment by purejosh** [comment]
Should probably rename to "Support for custom themes". Some spelling mistakes too. But I'll vote for this, because it's true. We need native theme support.

**Comment by lancepioch** [comment]
We'll be utilizing the Filament library. They already have a few themes available for download here: https://filamentphp.com/plugins

They also have a guide for making your own theme to customize your panel!
